x/tokenomics/module/tx_update_params_test.go

```diff
@@ -1,63 +0,0 @@
-package tokenomics_test
```

Comment:
Just confirming that this is deleted intentionally.

Reply:
Yes. This test was passing but for no particular reason. It didn't exercise anything that we actually care about and is now failing due to changes in the tokenomics params validation logic. It just doesn't make sense to keep it around.